samples/client/others/java/okhttp-gson-streaming/.openapi-generator/VERSION

```diff
@@ -1 +1 @@
-5.3.1-SNAPSHOT
\ No newline at end of file
+unset
```

Comment:
Can you please run these commands and commit the changes? 
./mvnw clean package 
./bin/generate-samples.sh
It will eliminate these Verdi on changes

Reply:
@spacether Please double check now

Reply:
These version files should have no changes
Are you running the commands as intelij Configurations? When I do that I get these unset values.
If you run the needed above commands from your shell it should revert these version file changes.
Please re-run from shell or revert these files.

Reply:
Hi, I've been running the commands in git bash on windows, I can try again.

Reply:
Update samples. See below.